helpers/fail2ban

```diff
@@ -100,6 +100,15 @@ ignoreregex =
     ynh_add_config --template="f2b_jail.conf" --destination="/etc/fail2ban/jail.d/$app.conf"
     ynh_add_config --template="f2b_filter.conf" --destination="/etc/fail2ban/filter.d/$app.conf"
 
+    # if "$logpath" doesn't exist (as if using --use_template argument), assign
+    # "$logpath" using the one in the package's conf/f2b_jail.conf template
+    if [ -z "$logpath" ]; then
+        # the first sed deleteds possibles spaces and the second one extract the path
+        logpath=$(grep logpath "$YNH_APP_BASEDIR/conf/f2b_jail.conf" | sed "s/ //g" | sed "s/logpath=//g")
+        # replace the '__APP__' by the real app name in the path
+        logpath=${logpath//__APP__/$app}
```

Comment:
maybe we could use something like [ynh_replace_vars](https://github.com/YunoHost/yunohost/blob/5746b94dff4db4fc7acb12586f1a8e4fab685a18/helpers/utils#L520) there? so that it works with any app variable?

Reply:
I tried something in https://github.com/YunoHost/yunohost/pull/1743/commits/5f7a207f112bae91cf61192cbeae3c9e631db6b0